### ISSUE TITLE
Implement character for /x/ in Luniksian

### ISSUE THREAD

**Issue by JosieOnGit** [BlackCatMS/Luniks#1]
Letter "x" is currently used and co-exists with "ś" in the sense of "Angliśká / Anglixká". For loanwords containing the letter "kh" or "ch", which make a /x/ sound, there is no character to represent that in the Luniksian script by itself. 

Room for thought:
- Is there actually a new character needed for /x/? Perhaps it's possible to translate the sound to a basic /k/ instead?
- Which native words or loan words used in the language even contain /x/?

**Comment by JosieOnGit** [comment]
Is there even a separate character needed for /x/? As far as readability goes, I think it'd be fine to continue using the existing alphabet with phonetics and not create a new character. This would leave the ambiguity of the sharp "s" sound using both ś/x in the Latin script. 

As far as the Luniksian alphabet goes, would it be possible to add a diacritic above its letter "k"? Is that needed? It would differentiate its sounds between /k/ and /x/ yet remove the need of creating a new letter altogether. Adding a curve above the letter, similarly to á and í, seems to work fine to me.

In short:
- Create diacritic for the Luniksian letter "k" by adding a curl above its character to indicate the /x/ sound from /k/
- Find out how to romanise that letter and sound. Would we write it as "kh" similarly to writing the Russian "x" in Latin?
- "Lakh" would be written as "Lak" with an accented "k" in Luniksian, be romanised as "Lakh" and pronounced as /ləx/

**Comment by JosieOnGit** [comment]
In commit 54b5bcd6a097f5db3132876770eaf69ef2213dba, a description clarifying the /x/ sound has been added to the language file.
